### ISSUE TITLE
When reviewing code, don't lose comments when escape or cancel is hit.

### ISSUE THREAD

**Issue by dylang** [isaacs/github#14]
Oh it's so painful when writing a long detailed code review comment and then accidentally hitting escape and finding out it's gone forever. Cache that text and put a little "Draft" marker on the page when somebody closes the comment form by accident.

**Comment by aspiers** [comment]
I agree.  The [Typio Form Recovery extension](https://chrome.google.com/webstore/detail/typio-form-recovery/djkbihbnjhkjahbhjaadbepppbpoedaa) is a great workaround which has saved my life on several occasions.
